pyispyb/filters.py

```diff
@@ -10,6 +10,20 @@ def session(
     return session
 
 
+def proposal(
```

Comment:
maybe prefix those with something like "query" ("proposal" -> "queryProposal") to make it clear that we are adding a query parameter when adding a dependency to this.

Reply:
I'm gonna put this in the future refactoring bag too, sorry https://github.com/ispyb/py-ispyb/issues/216